extensions/interactions/SetInput/directives/oppia-interactive-set-input.directive.ts

```diff
@@ -88,6 +91,10 @@ angular.module('oppia').directive('oppiaInteractiveSetInput', [
                   'I18N_INTERACTIONS_SET_INPUT_ADD_ITEM')
               }
             };
+            if (ctrl.buttonText) {
+              ctrl.schema.ui_config.add_element_text = $translate.instant(
```

Comment:
@seanlip Addressing your [comment](https://github.com/oppia/oppia/pull/8627#discussion_r378613555) from the previous PR.
I tested it locally and it worked fine for English. How do I test it for other languages?

Reply:
I think you can set your Preferred Site Language to something else in the Preferences page.

Reply:
@seanlip I checked and it wasn't working on other languages (mb, should've checked from the start).
I see that it needs i18n additions to work, and as this text will be dynamic, I've removed the translate call.